fixtures/packages/polymer/expected/test/smoke/html-tag.html

```diff
@@ -16,7 +16,7 @@
 <body>
   <script type="module">
 import { Element } from '../../polymer-element.js';
-import { html as html$0 } from '../../polymer.js';
+import { html as html$0 } from '../../lib/utils/html-tag.js';
```

Comment:
Add issue to get this to import from polymer.js

Reply:
Tracking as #336